dev-client/src/translations/en.json

```diff
@@ -473,10 +473,16 @@
         "body": "Data that has been entered at this depth will be deleted. This action cannot be undone.",
         "action": "Change"
       },
+      "delete_modal": {
+        "title": "Delete depth?",
+        "body": "Data that has been entered at this depth will be deleted. This action cannot be undone.",
+        "action": "Delete"
+      },
       "error": {
         "overlaps": "Overlaps existing",
         "ordering": "Interval bounds error"
       },
+      "bounds_unitless": "{{start}}-{{end}}",
       "bounds": "{{start}}-{{end}}{{units}}",
```

Comment:
```suggestion
      "bounds": "{{start}}–{{end}}{{units}}",
```

Reply:
@paulschreiber could you explain more why the use of the minus sign unicode codepoint and not the minus-hyphen?

Reply:
I replaced the hyphen with an en dash. En dashes are used to separate ranges of values. https://practicaltypography.com/hyphens-and-dashes.html